src/renderer/screens/platform/TwitterBanner.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from "react";
+import styled from "styled-components";
+import Box from "~/renderer/components/Box/Box";
+import Alert from "~/renderer/components/Alert";
+import Text from "~/renderer/components/Text";
+import { useTranslation } from "react-i18next";
+
+const Container: ThemedComponent<{}> = styled(Box).attrs(p => ({
+  mb: 24,
+}))``;
+
+export default function TwitterBanner() {
+  const { t } = useTranslation();
+
+  const params = {
+    text: t("platform.catalog.twitterBanner.tweetText"),
+    hashtags: "LedgerLiveApp",
+  };
+
+  const paramsURI = Object.entries(params).reduce(
+    (acc, [key, val]) => (acc += `&${key}=${encodeURI(val)}`),
+    "",
+  );
+
+  const url = `https://twitter.com/intent/tweet?${paramsURI}`;
```

Comment:
That works ! I see two easy improvements ! :) 

You could use the URL api to handle the query params ina mor
You could wrap the whole thing in a useMemo to prevent recalculation at every render !

Here's an extract of a similar use case from the same codebase:

`  const url = useMemo(() => {
    const urlObj = new URL(manifest.url.toString());

    urlObj.searchParams.set("backgroundColor", theme.background.paper);
    urlObj.searchParams.set("textColor", theme.text.shade100);

    return urlObj;
  }, [manifest.url, theme]);
`